checklist.md

```diff
@@ -0,0 +1,53 @@
+## F3 Activation Procedure (FRC-0099)
+
+This document outlines the procedure for activating F3 on the Filecoin network, adhering to FRC-0099. The process involves a multi-step approach, leveraging a smart contract to ensure transparency and community consensus.
+
+1.  **Parameter Selection and Bootstrap Epoch Determination**
+    *   Goal: Establish the "good parameter set" for F3 and the initial epoch for its activation.
+    *   Action: Analyze nv25 passive testing results. Consider network performance, stability, and community input.
+    *   Output: A finalized set of F3 parameters and a proposed bootstrap epoch.
+
+2.  **Manifest Definition and Storage**
+    *   Goal: Formally define the parameters and bootstrap epoch for distribution.
+    *   Action:
+        *   Create a manifest file (e.g., JSON) that clearly defines all F3 parameters and the selected bootstrap epoch.
+        *   Commit the manifest file into the `f3-activation-contract` repository.
+    *   Output: A verifiable manifest file stored in the repository.
+    *   **Manifest Link:** \[*Link to manifest file in repository*]
+
+3.  **Communication and Coordination**
```

Comment:
I think with the modification above that this step can be removed?

Reply:
I think I agree